src/main/java/edu/kis/powp/jobs2d/command/visitor/RectangleCanvas.java

```diff
@@ -0,0 +1,19 @@
+package edu.kis.powp.jobs2d.command.visitor;
+
+
+import edu.kis.powp.jobs2d.drivers.transformation.Point;
+
+public class RectangleCanvas implements Canvas {
```

Comment:
BTW: it could be a nice feature to be able to return a command drawing canvas boundaries.